src/program/media/models.py

```
@@ -0,0 +1,319 @@
+                case "720p" | "720i":
+                    resolution_height = 720
+                case "480p" | "480i":
+                    resolution_height = 480
+
+        bit_depth = None
+        if parsed_data.get("bit_depth"):
+            bd = parsed_data.get("bit_depth")
+            if bd:
+                try:
+                    bit_depth = int(bd.replace("bit", ""))
+                except:
```

Ruff (0.14.3)

Do not use bare `except`

(E722)